docs/src/pages/[platform]/theming/style-props/examples/PositionExamples.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, View, useTheme } from '@aws-amplify/ui-react';
+
+// use any CSS value
+export const PositionStylePropExample = () => {
+  return (
+    <View position="relative" width="100%" height="4.5rem">
+      <Button position="absolute" right="2rem" top="2rem">
```

Comment:
nit: make the button a little closer to the box
```suggestion
      <Button position="absolute" right="0.5rem" top="0.5rem">
```